cloudscape-design/components
# fix: restore focus delegation to split panel in visual-refresh-toolbar

### Description


- works for both spit panel positions
- set focus to split panel resize button when split panel opened via toolbar trigger button
- set focus back to split panel trigger button on the toolbar once it is closed from the close button on the split panel
-  returns split panel interactions in integ tests



Related links, issue #, if available: n/a

### How has this been tested?
- update to app layout integ tests where:
   - removed redundant classes
   - moved test added testutil functions and updated the tests to use those util functions
   - separated tooltip tests into multiple files with a util




got to deployment with the following path & params - /light/app-layout/with-split-panel?appLayoutWidget=true
- open split panel via the first button in the toolbar
- close split panel via its close button
- note the focus
Confirm /light/app-layout/with-split-panel also works in classic and visual refresh

- compare to current https://d21d5uik3ws71m.cloudfront.net/components/3f188c630a8446f5776683b9dffc0f2b6ce157ef/dev-pages/index.html#/light/app-layout/with-split-panel?appLayoutWidget=true

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_
- _Changes are covered with new/existing integration tests?_
</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
